lucet-builtins/Makefile

```diff
@@ -17,6 +17,7 @@ build/%.o: src/%.c
 	$(CC) $(COMMON_CFLAGS) -c $^ -o $@
 
 build/libbuiltins.so: $(LIBBUILTINS_OBJS)
+	$(CC) -Wl,-U,_lucet_vmctx_current_memory -Wl,-U,_lucet_vmctx_get_heap -shared $^ -o $@ 2> /dev/null || \
 	$(CC) -shared $^ -o $@
```

Comment:
I think you meant to delete the second invocation of CC

Reply:
This is intentional - the second invocation of `CC` is called if the first one fails (notice the `||` operator between them). Declaring only specific symbols as weak doesn't work on all llvm versions.

Reply:
Oh, nice. I totally missed the `||`. Could you put a comment explaining this? Its not intuitive, but quite smart.

Reply:
Added.